forage-android/src/main/java/com/joinforage/forage/android/network/TokenizeCardService.kt

```diff
@@ -16,23 +16,24 @@ internal class TokenizeCardService(
     private val httpUrl: HttpUrl,
     okHttpClient: OkHttpClient
 ) : NetworkService(okHttpClient) {
-    suspend fun tokenizeCard(cardNumber: String): ForageApiResponse<String> = try {
-        tokenizeCardCoroutine(cardNumber)
+    suspend fun tokenizeCard(cardNumber: String, userId: String? = null): ForageApiResponse<String> = try {
+        tokenizeCardCoroutine(cardNumber, userId)
     } catch (ex: IOException) {
         ForageApiResponse.Failure(listOf(ForageError(500, "unknown_server_error", ex.message.orEmpty())))
     }
 
-    private suspend fun tokenizeCardCoroutine(cardNumber: String): ForageApiResponse<String> {
+    private suspend fun tokenizeCardCoroutine(cardNumber: String, userId: String?): ForageApiResponse<String> {
         val url = getTokenizeCardUrl()
 
         val requestBody =
-            PaymentMethodRequestBody(cardNumber = cardNumber).toJSONObject().toString()
+            PaymentMethodRequestBody(cardNumber = cardNumber, userId = userId).toJSONObject().toString()
 
         val body: RequestBody =
             requestBody.toRequestBody("application/json; charset=utf-8".toMediaTypeOrNull())
 
         val request: Request = Request.Builder()
             .url(url)
+            .header(ForageConstants.Headers.API_VERSION, "2023-03-31")
```

Comment:
Version header may do better in the global part of the http client, see here: https://github.com/teamforage/forage-android-sdk/blob/31ea5a82cf20bc0e6b3c65f2263f43f8951e55c4/forage-android/src/main/java/com/joinforage/forage/android/network/OkHttpClientBuilder.kt#L17

Reply:
Addressed here: a480072

Reply:
Should this request stay at version: "2023-02-01"?

https://github.com/teamforage/forage-android-sdk/blob/31ea5a82cf20bc0e6b3c65f2263f43f8951e55c4/forage-android/src/main/java/com/joinforage/forage/android/network/MessageStatusService.kt#L25

Reply:
@ForageRob unless we determined we are changing how our versioning currently works, we can't pass the header globally. All endpoints without a version `2023-03-31` will fail. Did we decide to update this?

@djoksimo that endpoint still needs the version header. It is an internal endpoint that I versioned a while ago to change the response body structure.

Reply:
Reverted/dropped the commit so now only the `POST payment_methods` has this API_VERSION attached